### DISCUSSION TITLE
Chromium version 127.0.6533.72 [UPDATE]

### DISCUSSION THREAD

**Discussion by MartinVonReichenberg** [nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt#190] [Ideas]
Hello,

There has been a new [update of Chromium](https://chromiumdash.appspot.com/releases?platform=Linux), please update the `nwjs-ffmpeg-prebuilt` with a new release.

Thanks!

**Comment by zkrige** [comment]
Hi @MartinVonReichenberg - nwjs-ffmpeg-prebuilt is based off nwjs, and we can only update when nwjs releases a new version

**Reply by MartinVonReichenberg** [reply]
> Hi @MartinVonReichenberg - nwjs-ffmpeg-prebuilt is based off nwjs, and we can only update when nwjs releases a new version


Ahh, wrong repository, wrong thread then. 

My bad . . .
